bootstrap/src/main/kotlin/jp/co/soramitsu/bootstrap/controller/IrohaController.kt

```diff
@@ -1,24 +1,39 @@
 package jp.co.soramitsu.bootstrap.controller
 
+import com.fasterxml.jackson.databind.ObjectMapper
 import jp.co.soramitsu.bootstrap.dto.*
+import jp.co.soramitsu.bootstrap.exceptions.ErrorCodes
 import jp.co.soramitsu.bootstrap.genesis.GenesisInterface
 import jp.co.soramitsu.crypto.ed25519.Ed25519Sha3
 import mu.KLogging
 import org.springframework.http.HttpStatus
 import org.springframework.http.ResponseEntity
 import org.springframework.web.bind.annotation.*
+import org.testcontainers.shaded.javax.ws.rs.QueryParam
+import java.util.stream.Collectors.toList
+import javax.validation.constraints.NotNull
 import javax.xml.bind.DatatypeConverter
 
 @RestController
 @RequestMapping("/iroha")
 class IrohaController(val genesisFactories: List<GenesisInterface>) {
 
     private val log = KLogging().logger
+    private val mapper = ObjectMapper()
 
-    @GetMapping("/config/accounts/{project}/{env}")
-    fun getNeededAccounts(@PathVariable("project") project: String, @PathVariable("env") env: String): ResponseEntity<List<AccountPrototype>> {
+    @GetMapping("/config/accounts/{project}/{env}/{peersCount}")
+    fun getNeededAccounts(
+        @PathVariable("project") project: String,
+        @PathVariable("env") env: String,
+        @PathVariable("peersCount") peersCount: Int
+    ): ResponseEntity<NeededAccountsResponse> {
+        if (peersCount == null || peersCount == 0) {
```

Comment:
But `peersCount` cannot be nullable. Or am I wrong?

Reply:
fixed